docs/2.7.0/docs/deploy-daml/infrastructure-architecture/high-availability/basic-ha/overview.rst

```diff
@@ -8,23 +8,17 @@ Overview
 
 HA for Daml solutions focuses on the following components running in separate processes:
 
-* Participant nodes:
-
-    * Ledger API server.
-        * gRPC server.
-        * SQL indexer.
-
-    * Sync service.
+* Participant nodes
 
 * Domains:
 
-    * Sequencer.
+    * Sequencer
 
-    * Mediator.
+    * Mediator
 
-    * Topology manager.
+    * Domain manager
 
-    * Domain service.
+    * Domain service
```

Comment:
this component does not exist anymore, you can remove it

Reply:
Same as above, I'll incorporate this in the next PR.